ci/run-docker.sh

```diff
@@ -43,12 +43,13 @@ run() {
         --user "$(id -u)":"$(id -g)" \
         --env LIBC_CI \
         --env LIBC_CI_ZBUILD_STD \
+        --env RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS64 \
         --env CARGO_HOME=/cargo \
         --env CARGO_TARGET_DIR=/checkout/target \
         --volume "$CARGO_HOME":/cargo \
         --volume "$(rustc --print sysroot)":/rust:ro \
-        --volume "$(pwd)":/checkout:ro \
-        --volume "$(pwd)"/target:/checkout/target \
+        --volume "$PWD":/checkout:ro \
+        --volume "$PWD"/target:/checkout/target \
```

Comment:
Is this just style (fine if so) or did some platform not work with PWD?

Reply:
Not just style, running the script under `set -x` will give prints for the execution of the `$(pwd)` subprocess.  
At the moment I was having a hard time getting the CI to do what I wanted and the extra prints were annoying.